src/react/hooks/__tests__/useSuspenseQuery.test.tsx

```diff
@@ -298,14 +298,53 @@ describe('useSuspenseQuery', () => {
       });
     }).toThrowError(
       new InvariantError(
-        'Could not find a "suspenseCache" in the context. Wrap the root component ' +
-          'in an <ApolloProvider> and provide a suspenseCache.'
+        'Could not find a "suspenseCache" in the context or passed in as an option. ' +
+          'Wrap the root component in an <ApolloProvider> and provide a suspenseCache, ' +
+          'or pass a SuspenseCache instance in via options.'
       )
     );
 
     consoleSpy.mockRestore();
   });
 
+  it('also accepts a `suspenseCache` option', () => {
```

Comment:
```suggestion
  it('does not throw when provided a `suspenseCache` option', () => {
```

[nit] Since it looks like this test is checking to make sure no error is thrown, I think it makes sense for the test name to reflect that.